LayoutTests/svg/text/font-small-enlarged-minimum-larger-expected.svg

```diff
@@ -1,7 +1,4 @@
 <svg xmlns="http://www.w3.org/2000/svg" style="width: 400px; height: 400px;">
-  <script>
-    internals.settings.setMinimumFontSize(80.0);
```

Comment:
You're not supposed to change the test -- it no longer tests "font-small-enlarged-minimum"?
If you leave the test as it is you'll get the 0.01% diff in the reftest resuts, after your patch?